subsys/net/lib/fota_download/Kconfig

```diff
@@ -56,6 +56,12 @@ config FOTA_DOWNLOAD_RESOURCE_LOCATOR_LENGTH
 	int "Size of buffer used for Resource locator"
 	default 512
 
+config FOTA_DOWNLOAD_SEC_TAG_LIST_SIZE_MAX
```

Comment:
We already have `NRF_SOCKET_TLS_MAX_SEC_TAG_LIST_SIZE` which is the maximum size supported by modem.

Reply:
Yeah, maybe I should set an allowed range when TLS in modem is used. Not sure how to do that, though

Reply:
```
default NRF_SOCKET_TLS_MAX_SEC_TAG_LIST_SIZE  if NRF_MODEM_LIB
```

Reply:
Yes, that one works if TLS stack in the modem is used. But if TLS is ran in the app core, `NRF_SOCKET_TLS_MAX_SEC_TAG_LIST_SIZE` is not relevant anymore. I think I'm going to leave the deafult at 5 as an okay trade-off between RAM usage and large-enough-for-most-users